webcomponents/vectortile-view-component/src/index.html

```diff
@@ -25,10 +24,54 @@
     flex-grow: 1;
   }
 
+  app-legend-view {
```

Comment:
Formatting. Of misschien [Prettier](https://dev.to/this-is-angular/configure-prettier-and-eslint-with-angular-526c) toevoegen.